htdocs/js/jquery.talkform.js

```diff
@@ -0,0 +1,96 @@
+// Helpers specific to the talkform aka slowreply aka ReplyPage aka The Varmint
+// See also jquery.quickreply.js, jquery.replyforms.js
+
+jQuery(function($){
+
+    var talkform = $('#postform');
+    var fromOptions = $('input[name="usertype"]');
+    var loginForms = $('.from-login');
+    var iconSelect = $('#prop_picture_keyword');
+
+    // Reset text inputs and checkboxes to their initial values. Inputs can
+    // specify their initial value with the data-revert-value attribute,
+    // otherwise we assume empty/false.
+    jQuery.fn.extend({
+        resetFormFields: function() {
+            this.find('input').each(function(i, elm){
+                var type = elm.getAttribute('type');
+                if (type === 'checkbox' || type === 'radio') {
+                    elm.checked = ($(elm).data('revertValue') || false);
```

Comment:
Wait... this might be totally unnecessary. The parent form element’s reset() method might do this automatically. Need to test to make sure it does what I’m thinking, tho.

Reply:
Past Nick is half-right. reset()'s off the table because these aren't sub-forms and we don't want to whack the whole comment, but input.defaultValue and input.defaultChecked still let me skip that data element BS. nice.